.eslintrc

```diff
@@ -1,3 +1,26 @@
 {
-  "extends": "seek"
+  "extends": "seek",
+  "overrides": [
+    {
+      "files": ["*.{js,ts,tsx}"],
+      "excludedFiles": [
+        "*.{docs,gallery,screenshots,stories}.tsx",
+        "site/**/*.{ts,tsx}"
+      ],
+      "rules": {
+        "no-restricted-imports": [
+          "error",
+          {
+            "patterns": [
+              "braid-design-system**",
+              "!braid-design-system/lib/utils/source.macro",
+              "braid-src/**",
+              "site/**",
+              "**/site/**"
```

Comment:
Is this still needed in addition to `site/**`?

Reply:
Yes, this one is to restrict imports from the `site` directory, i.e. `../../../site/etc`.
The one on line 18 is to match imports from the `site` alias, i.e. `site/etc`.